columnphysics/icepack_therm_itd.F90

```diff
@@ -884,13 +884,13 @@ subroutine lateral_melt (dt,         ncat,       &
                                fresh,      fsalt,      &
                                fhocn,      faero_ocn,  &
                                fiso_ocn,               &
-                               rside,      meltl,      &
-                               fside,      wlat,       &
+                               rsiden,      meltl,     &
```

Comment:
fix column alignment (picky, sorry, but might as well fix it now)

Reply:
The diffs were so big with icepack_therm_itd.F90 I did not see your comments!